packages/squiggle-lang/src/fr/system.ts

```diff
@@ -0,0 +1,32 @@
+import { makeDefinition } from "../library/registry/fnDefinition.js";
+import { frNumber } from "../library/registry/frTypes.js";
+import { FnFactory } from "../library/registry/helpers.js";
+import { ImmutableMap } from "../utility/immutableMap.js";
+import { Value, vString } from "../value/index.js";
+
+// automatically updated on release by ops/ patch-js utils
```

Comment:
TODO: I realized we'd need to change ``patch-js``

Reply:
Can you just move it back to `version.ts` for now? I could update `patch-js`, but it's better if the patched file is small, because it will be reformatted on patch and Babel tends to ignore the whitespace.

Reply:
Sure, can do.